server/src/services/api/api-chart.ts

```diff
@@ -0,0 +1,16 @@
+import environments from '../../environments/environment';
+const stringifyObject = require('stringify-object');
+import {ChartModel} from '../../models/chart.models'
+
+export function getCovidTrends(inputData: ChartModel) {
+
+    const praparedInput =  
+        stringifyObject(
+            inputData, {
+                indent: '',
+                singleQuotes: false
+            }
+    );
+
+    return encodeURI(environments.CHARTSAPI_URL + '?c='+ praparedInput);
```

Comment:
Urls are pretty long.
And shorting it with CHARTSAPI is not free of charge.
I'm going think of how to improve the approach. But it definitely will come in one the future PRs

Reply:
it's pitty 🌵